test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Stubs/TempDbDataService/TempDbDataService.cs

```diff
@@ -134,7 +134,7 @@ private static EntityConnection GetConnectionForType(BaseModelType type)
             if (!connections.ContainsKey(type) || connections[type] == null)
             {
                 string dbName = Enum.GetName(typeof(BaseModelType), type);
-                string dataSource = DataUtil.DefaultDataSource;
+                string dataSource = DataUtil.DefaultDataSource;

```

Comment:
Maybe this was not meant for checkin?

Reply:
True. No idea what the change means.

Reply:
Looks like this removes an empty trailing space, which CA would have complained about.  So it's good clean-up in order for the check-in to pass, even if it's not material to the PR.

---
In reply to: [385690196](https://github.com/OData/odata.net/pull/1656#discussion_r385690196) [](ancestors = 385690196)